simple-rest-server/get.go

```diff
@@ -3,12 +3,40 @@ package main
 import (
 	"fmt"
 	"net/http"
-
+    "os"
+	"encoding/csv"
 	"github.com/julienschmidt/httprouter"
 )
 
 func ListTask(respWriter http.ResponseWriter, request *http.Request, params httprouter.Params) {
-	fmt.Fprintf(respWriter, "Show Details for ToDo")
+	 csvfile, err := os.Open("tasks.csv")
+
+         if err != nil {
+                 fmt.Println(err)
+                 return
+         }
+
+         defer csvfile.Close()
+
+         reader := csv.NewReader(csvfile)
+
+         reader.FieldsPerRecord = -1 
+         
+         rawCSVdata, err := reader.ReadAll()
+
+         if err != nil {
+                 fmt.Println(err)
+                 os.Exit(1)
+         }
+
+         // sanity check, display to standard output
+         for _, each := range rawCSVdata {
+             status := "unchecked"
+             if each[1] == "true" {  // each[1] will be the status - either true or false
```

Comment:
> we should use the build-in strings.Compare() 
I don't think so. `if each[1] == "true"` is a build-in string comparaison.

Either 0/1 "false"/"true" is fine. We're reading a text file anyway.